packages/driver/src/cy/listeners.js

```diff
@@ -51,6 +52,17 @@ module.exports = {
 
     // set onerror global handler
     contentWindow.onerror = callbacks.onError
+    contentWindow.addEventListener('unhandledrejection', ({ reason }) => {
```

Comment:
since this is going to be moved to its own `unhandled:rejection` event, you can probably move this code to `cy.js` `contentWindowListeners`, i think